src/utils/code.ts

```diff
@@ -365,6 +365,22 @@ const getIconsImports = (components: IComponents) => {
   })
 }
 
+export const generateMainTsx = (params: any, fileName: string) => {
+  let refsCode = ``
+  let appCode = `return <${fileName}App \n`
+  params.map((param: any) => {
```

Comment:
https://stackoverflow.com/questions/62894571/join-map-values-into-single-string

Reply:
For this I'll first make an array of all values and then join them. Therefore iterating twice rather than once.

Reply:
the list is very small, so performance wise it would be a negligible difference. The change I suggested for readability

Reply:
There are a lot of places where we can optimize for readability. For now, we are focussing more on functionality rather than readability, edge cases, exceptional handling, performance, etc.